packages/flutter_tools/test/general.shard/ios/xcodeproj_test.dart

```diff
@@ -12,7 +12,6 @@ import 'package:flutter_tools/src/base/terminal.dart';
 import 'package:flutter_tools/src/build_info.dart';
 import 'package:flutter_tools/src/ios/xcodeproj.dart';
 import 'package:flutter_tools/src/project.dart';
-import 'package:flutter_tools/src/reporting/reporting.dart';
```

Comment:
Looks like one got away.
```
packages/flutter_tools/lib/src/ios/xcodeproj.dart:297:20: Error: Method not found: 'Usage.test'.
      usage: Usage.test(),
                   ^^^^
```

Reply:
Ahh, I had move this back from test/src to reporting since the usage is also not in test.

I did add some visibleForTesting annotations...